srcalc/src/main/java/gov/va/med/srcalc/domain/model/Rule.java

```diff
@@ -101,7 +103,7 @@ void setId(final int id)
         return fMatchers;
     }
     
-    void setMatchers(final List<ValueMatcher> matchers)
+    public void setMatchers(final List<ValueMatcher> matchers)
```

Comment:
As I mentioned in a previous review, offering a getter and a setter for a mutable collection makes for a confusing interface. You do need `setMatchers()` for Hibernate's bean construction, but I recommend keeping it package-private.